### PR TITLE
chore(ui): align prettier + eslint

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] JIRA -[LW-9997](https://input-output.atlassian.net/browse/LW-9997)
- [ ] ~~Proper tests implemented~~
- [ ] ~~Screenshots added.~~

---

## Proposed solution

use eslint-plugin-prettier to fix conflicting rulese between prettier and easlint

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

[LW-9997]: https://input-output.atlassian.net/browse/LW-9997?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -92,18 +92,19 @@
...
     "eslint-plugin-max-params-no-constructor": "^0.0.4",
     "eslint-plugin-prefer-arrow-functions": "^3.1.4",
+    "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react": "7.31.8",
     "eslint-plugin-storybook": "^0.6.11",
     "eslint-plugin-unicorn": "^45.0.2",
     "http-server": "^14.1.1",
     "jest-image-snapshot": "^6.1.0",
-    "prettier": "^2.4.1",
+    "prettier": "3.2.5",
```
Comment (mchappell) [inline packages/ui/package.json:107]: should also align in the root `package.json` for compatability, or simply hoist to root level

File: packages/ui/src/design-system/table/table-row.component.tsx [hunk 2]
```diff
@@ -17,12 +17,9 @@ export interface RowProps<T extends object> {
   selected?: boolean;
   onSelect?: () => void;
   onClick?: () => void;
-  // TODO fix conflicting prettier and eslint configuration in @lace/ui; https://input-output.atlassian.net/browse/LW-9997
```
Comment (przemyslaw-wlodek) [inline packages/ui/src/design-system/table/table-row.component.tsx:20]: 🚀

**Review by przemyslaw-wlodek — APPROVED** [review]
Thanks Vit! 🙌🏻 It'd be great if we could align eslint/prettier-related packages in the root (as @mchappell mentioned) but it may be a topic for a separate ticket.

File: packages/ui/package.json (hunk 1 above)
Comment (vetalcore) [reply packages/ui/package.json:107]: i've run into multiple issues while doing that coming from `@atixlabs/eslint-config/configurations/react`, so i'd rather consider this out of scope of this one.

**Review by mchappell — APPROVED** [review]

**Review by przemyslaw-wlodek — APPROVED** [review]

**Pull request merged**